libethereum/Executive.cpp

```diff
@@ -163,7 +163,13 @@ void StandardTrace::operator()(uint64_t _steps, uint64_t PC, Instruction inst, b
 
 string StandardTrace::json(bool _styled) const
 {
-    return _styled ? Json::StyledWriter().write(m_trace) : Json::FastWriter().write(m_trace);
+	if (_styled)
+		return Json::StyledWriter().write(m_trace);
+
+	return std::accumulate(std::next(m_trace.begin()), m_trace.end(), Json::FastWriter().write(m_trace[0]),
```

Comment:
You have to include `accumulate` function, probably by `#include <algorithm>`.

Reply:
Done, also rebased